src/astarte-client/models/Mapping/index.ts

```diff
@@ -152,6 +152,18 @@ class AstarteMapping {
   static toJSON(mapping: AstarteMapping): AstarteMappingJSON {
     return toAstarteMappingDTO(mapping);
   }
+
+  static getEndpointParameters(endpoint: string): string[] {
+    const pathParamsArray: string[] = [];
+    const endpointParams = endpoint.match(/%\{(.+?)\}/g);
```

Comment:
Similar to `mappingEndpointRegex` and `mappingEndpointParamRegex` this one should be defined at the beginning of the file